server.js

```diff
@@ -0,0 +1,4 @@
+const express = require("express")
+
+const app = express();
+
```

Comment:
```suggestion
app(80, () => { console.log("app is running on port 80"))
```
You forgot to add ports